portfolio/src/main/webapp/style.css

```
@@ -1,11 +1,34 @@
+body {
+    background-color: #937691; /* Light Purple */
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
-  color: #FEE9E1
+  font-family: Georgia;
+  color: #FEE9E1; /* Light Peach */
 }
 
-#random-fact-container {
-  margin-top: 20px;
+#content img {
+  width: 200px;
+  height: 200px;
+}
+
+h1, h2 {
```

nit: separate selectors with a newline (https://docs.google.com/document/d/16mJI_KySB5wexG0c7yyxQGjrJNDzVEr3twDHt_IXHzg/edit#)
```suggestion
h1, 
h2 {
```